### DISCUSSION TITLE
How do I get error logs related to emails?

### DISCUSSION THREAD

**Discussion by uSkizzik** [makeplane/plane#2867] [Q&A]
Hi! So, I'm having some issues where inviting a person doesn't seem to send an email at all. I'm self-hosting my mail service and it works with my other tools. I've experienced issues with it in the past so I'm wondering where I can find the error logs for emails so I can debug the issue.

**Comment by pablohashescobar** [comment]
You can enable debug logs by setting the `DEBUG=1` for the api container.

**Reply by uSkizzik** [reply]
I don't think that worked back when I was trying to get this to work.
Any other ideas?

**Comment by vihar** [comment]
@uSkizzik, taking this conversation to, https://github.com/makeplane/plane/issues/2878.
